framework/test/mocha/unit/modules/chain/components/storage/entities/round_delegates.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable mocha/no-pending-tests */
+/*
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const BigNum = require('@liskhq/bignum');
+const { StorageSandbox } = require('../../../../../../common/storage_sandbox');
+const seeder = require('../../../../../../common/storage_seed');
+
+describe('RoundDelegates', () => {
+	let adapter;
+	let storage;
+	let RoundDelegatesEntity;
+	let SQLs;
+
+	before(async () => {
+		storage = new StorageSandbox(
+			__testContext.config.components.storage,
+			'lisk_test_storage_custom_round_delegates_chain_module',
+		);
+		await storage.bootstrap();
+
+		adapter = storage.adapter;
+		RoundDelegatesEntity = storage.entities.RoundDelegates;
+		SQLs = RoundDelegatesEntity.SQLs;
+	});
+
+	beforeEach(() => seeder.seed(storage));
+
+	afterEach(() => {
+		sinonSandbox.restore();
+		return seeder.reset(storage);
+	});
+
+	describe('summedRound()', () => {
```

Comment:
I see that it's maybe more straight forward to write tests in mocha. But would be better if we could write this tests in jest. Maybe create an issue for later to migrate it. Or if it's not a big effort do it in this PR ?

Reply:
In this case I just renamed this file from `rounds.js` to `round_delegates.js` :)